nebula/docker/nebula-production.yml

```diff
@@ -0,0 +1,41 @@
+services:
+  transcriber:
+    extends:
+      file: nebula.yml
+      service: transcriber
+    image: ghcr.io/ls1intum/edutelligence/nebula-transcriber:${NEBULA_DOCKER_TAG}
+    pull_policy: always
+    restart: unless-stopped
+    container_name: transcriber-service
+    volumes:
+      - ${NEBULA_LLM_CONFIG_FILE}:/app/llm_config.nebula.yml:ro
+      - ${NEBULA_TEMP_DIR}:/app/temp
+    environment:
+      LLM_LOG_LEVEL: DEBUG
+
+  faq:
+    extends:
+      file: nebula.yml
+      service: faq
+    image: ghcr.io/ls1intum/edutelligence/nebula-faq:${NEBULA_DOCKER_TAG}
+    pull_policy: always
+    restart: unless-stopped
+    container_name: faq-service
+    volumes:
+      - ${NEBULA_LLM_CONFIG_FILE}:/app/llm_config.nebula.yml:ro
+    environment:
+      LLM_LOG_LEVEL: DEBUG
+
+  nginx:
+    extends:
+      file: nebula.yml
+      service: nginx
+    restart: unless-stopped
+    container_name: nginx-proxy
+    volumes:
+      - ${NEBULA_NGINX_CONFIG_FILE}:/usr/local/openresty/nginx/conf/nginx.conf:ro
```

Comment:
In the production setup, make sure to expose ports 80 and 443, and add a mount for the SSL certificates.